.github/workflows/predeploy.yml

```diff
@@ -0,0 +1,58 @@
+name: Predeploy
+
+concurrency: ${{ github.ref }}
+
+on: 
+  workflow_dispatch:
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+      RESOURCE_GROUP_NAME: "aqovia-devbox"
+      IDENTITY_NAME: "aqovia-devcenter-identity"
+      GALLARY_NAME: "aqovia_compute_gallery"
```

Comment:
We should define the following variables here:
* VNET_NAME
* SUBNET_NAME
* STORAGE_ACCOUNT_NAME

Reply:
I think you have missed this comment. @iluchyk-beep